### PR TITLE
Capitalize “Slack”

### PR DESCRIPTION
**Pull request by mattstein**
Hi! I think Slack wants to be a proper noun so I’m offering this stunning one-character PR.

----
:books: Documentation preview :books:: https://writethedocs-www--1994.org.readthedocs.build/

### CODE REVIEW

**Comment by plaindocs** [comment]
I appreciate the full stunningness of it, but if you're gonna go there can I perhaps suggest:

```
❯ ack "slack " docs/ | wc
     27     741    6173
```

Using the space as a lazy proxy for "slack not in a URL?"

**Comment by mattstein** [comment]
Am I understanding correctly that you’re a higher-level wizard than me and you’re suggesting I be more thorough and fix all the `slack`s instead of just this one?

**Comment by plaindocs** [comment]
I'm not sure about the first half of your comment ;-), but yes I'm suggesting that if one is worth fixing so are the rest. And even that fixing one and not the rest is even worse than having them all lower case.

**Comment by mattstein** [comment]
I didn’t notice any other improperly-lowercased ones, but I also didn’t check every page. I can fix that, thank you!

**Comment by mattstein** [comment]
Fixed! Went through by hand to avoid wrecking anything, and the only case I wasn’t sure about (and left alone out of caution) was this: https://github.com/writethedocs/www/blob/main/docs/sponsorship/index.rst?plain=1#L44

Not sure what it’s doing, but it looks like a key and not a piece of text.

**Comment by plaindocs** [comment]
> Fixed! Went through by hand to avoid wrecking anything, and the only case I wasn’t sure about (and left alone out of caution) was this: https://github.com/writethedocs/www/blob/main/docs/sponsorship/index.rst?plain=1#L44
> 
> Not sure what it’s doing, but it looks like a key and not a piece of text.

Correct, it's including `slack.rst` from that directory in the table of contents. :+1:

**Review by plaindocs — APPROVED** [review]
Thanks for that. :+1: 

If you add a line to https://github.com/writethedocs/www/blob/main/vale/WTD/Branding.yml that looks like `slack: Slack` it should stop it from happening in the future. As it in will shout if it detects instances of `slack` in text.

**Comment by mattstein** [comment]
You’re welcome. :)

I [added Slack to that branding list](https://github.com/writethedocs/www/pull/1994/commits/e97c69e4f5872e4f617e3d55c2bbd5923b41cd1d) as you suggested.

> Correct, it's including `slack.rst` from that directory in the table of contents. 👍

I’m still not sure where to find said table of contents looking at the [Community Sponsorship](https://www.writethedocs.org/sponsorship/) page, or how I should be able to verify that the table of contents block is behaving like it’s supposed to. Not that it matters—just curious!

Thanks for taking the time to review this and encouraging me to be more thorough about it.

**Comment by plaindocs** [comment]
> You’re welcome. :)
> 
> I [added Slack to that branding list](https://github.com/writethedocs/www/pull/1994/commits/e97c69e4f5872e4f617e3d55c2bbd5923b41cd1d) as you suggested.
> 
> > Correct, it's including `slack.rst` from that directory in the table of contents. 👍
> 
> I’m still not sure where to find said table of contents looking at the [Community Sponsorship](https://www.writethedocs.org/sponsorship/) page, or how I should be able to verify that the table of contents block is behaving like it’s supposed to. Not that it matters—just curious!
> 
> Thanks for taking the time to review this and encouraging me to be more thorough about it.

Yeah, you can ignore it. It's in `/docs/sponsorship/index.rst`, right? It's referencing https://github.com/writethedocs/www/blob/main/docs/sponsorship/slack.rst which is in the same directory, and sphinx doesn't include the extension in the TOC. :man_shrugging:

**Comment by plaindocs** [comment]
Perfect, the check is working as expected. :-D 

```
{"message": "[WTD.Branding] Use 'Slack' instead of 'slack'", "location": {"path": "docs/conf/portland/2020/writing-day.rst", "range": {"start": {"line": 53, "column": 29}}}, "severity": "ERROR"}
{"message": "[WTD.Branding] Use 'Slack' instead of 'slack'", "location": {"path": "docs/organizer-guide/meetups/starting.rst", "range": {"start": {"line": 6, "column": 22}}}, "severity": "ERROR"}
{"message": "[WTD.Branding] Use 'Slack' instead of 'slack'", "location": {"path": "docs/include/slack.txt", "range": {"start": {"line": 3, "column": [24](https://github.com/writethedocs/www/actions/runs/5727948244/job/15522265602?pr=1994#step:4:25)6}}}, "severity": "ERROR"}
```

The [first one](https://github.com/writethedocs/www/pull/1994/files#diff-68852169702ddafc6a2cfde6df44a494fc356f2814c39fabd14ac9bd78513f32R53) is an odd one, because it *should* be ignoring  the code style slack. Probably it's checking it because it's a link.

You could either rewrite that link text, or surround the entire line with 

```
.. vale off

[link with slack in the text](example.com)

.. vale on
```

I've not looked at the other two.

**Comment by mattstein** [comment]
Thanks. Was just looking at those and will see if I can iron them out!

File: docs/include/slack.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Welcome to the Write the Docs Slack! We're glad you're here, and we look forward to your contributions to our community.
 
-We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our Slack guidelines. All this information is at <https://www.writethedocs.org/slack/> -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our guidelines are an important tool to further this goal!
+We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our Slack guidelines. All this information is on [our Slack page](https://www.writethedocs.org/slack/) -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our guidelines are an important tool to further this goal!
```
Comment (plaindocs) [inline docs/include/slack.txt:5]: I'm not sure if this is supposed to be in rst or markdown :-D, I'll try to track it down.

Comment (mattstein) [reply docs/include/slack.txt:5]: Oh gosh, that was pure instinct on my part—sorry!

File: docs/conf/portland/2020/writing-day.rst [hunk 2]
```diff
@@ -28,29 +28,29 @@ You've just been asked by your team to create documentation for your project. Bu
...
 
 We are kicking off a pilot to define best practices for `cross-domain management of glossaries <https://docs.google.com/document/d/1Fjrl34ErnYammel9WmvXJ3rMWFANjoSiiGyyNSYOXUg/>`_. We want help defining what an ideal glossary entry or preferred word list entry should look like. Work will happen in this `glossary entry <https://docs.google.com/document/d/1wsSLQ_T8skVdlvjF5ayZa5IhKbTazdqm97lHX0qc16Q>`_ page.
 
-Reach out to @alyssa and @cameronshorter on WTD slack if interested. (@cameronshorter is in the Australian timezone.)
+Reach out to @alyssa and @cameronshorter on WTD Slack if interested. (@cameronshorter is in the Australian timezone.)
 
 **General chit-chat**
 
-We hang out in `thegooddocs.slack.com <https://join.slack.com/t/thegooddocs/shared_invite/enQtODkyNjI5MDc0NjE0LTUyNGFiZmU1MjIzNDMwN2E3NmQwODQwZmRkYWI5MDhlMzdjYzg4Nzg4YjM3ODA0NGE4MTgyYzdkMGViMTI2MDM>`_.
+We hang out in [a group Slack workspace](https://join.slack.com/t/thegooddocs/shared_invite/enQtODkyNjI5MDc0NjE0LTUyNGFiZmU1MjIzNDMwN2E3NmQwODQwZmRkYWI5MDhlMzdjYzg4Nzg4YjM3ODA0NGE4MTgyYzdkMGViMTI2MDM).
```
Comment (plaindocs) [inline docs/conf/portland/2020/writing-day.rst]:
```suggestion
We hang out in [The Good Docs Slack workspace](https://join.slack.com/t/thegooddocs/shared_invite/enQtODkyNjI5MDc0NjE0LTUyNGFiZmU1MjIzNDMwN2E3NmQwODQwZmRkYWI5MDhlMzdjYzg4Nzg4YjM3ODA0NGE4MTgyYzdkMGViMTI2MDM).
```

**Comment by plaindocs** [comment]
This turned into a little more work than either of us was expecting I think! Thank you.

But it's good to go now, feel free to mash that big green merge button!

**Comment by mattstein** [comment]
You’re welcome @plaindocs, and I would be honored to mash the button but I do not have that power.

Screen Shot 2023-08-03 at 09 00 42 AM@2x

Maybe you meant that for another maintainer though?

**Comment by plaindocs** [comment]
Sorry about that @mattstein ! I thought that once approved anyone could mash. On this occasion I shall do it for you. :-)

**Comment by mattstein** [comment]
No problem, I felt like I was there mashing in spirit.

**Pull request merged**